Dockerfile.wasi-builder

```diff
@@ -1,16 +1,17 @@
-FROM ubuntu:latest@sha256:4b1d0c4a2d2aaf63b37111f34eb9fa89fa1bf53dd6e4ca954d47caebca4005c2
+ARG WASM_BASE=20220116
+FROM ghcr.io/vmware-labs/wasm-base:${WASM_BASE}
 ARG WASI_SDK_VERSION=19
 ENV WASI_SDK=wasi-sdk-${WASI_SDK_VERSION}
 ENV WASI_SDK_ROOT=/wasi-sdk
+ENV WASI_SDK_PATH=${WASI_SDK_ROOT}
 RUN apt update && \
       DEBIAN_FRONTEND=noninteractive apt install -y \
         autoconf \
         automake \
         build-essential \
         clang \
         git \
-        pkg-config \
-        wget
```

Comment:
Hm, is this working? Asking because wget is used in the RUN just below

Reply:
Yes, it goes now at the wasm-base container image: https://github.com/vmware-labs/webassembly-language-runtimes/pull/32/files#diff-bccac2ed304716d87d96d8aaa431fe937b865d064b8882fe90a953d25680e3baR5